<ui version="4.0" >
 <class>ProxyDia</class>
 <widget class="QDialog" name="ProxyDia" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="styleSheet" >
   <string>QDialog {
 
background-color:rgb(246,243,243);

}
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>190</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="connectMode" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>DirectConnect</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>HttpProxy</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Socks5Proxy</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>65</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>64</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>123</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>123</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>PassWord</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HostEdit" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>88</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortEdit" >
   <property name="geometry" >
    <rect>
     <x>158</x>
     <y>86</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserEdit" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>146</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWordEdit" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>145</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>6</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TestButton" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>30</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="Test_Show" >
   <property name="geometry" >
    <rect>
     <x>191</x>
     <y>7</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProxyDia</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProxyDia</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
